docs/atomic-host.1.md

```diff
@@ -34,8 +34,18 @@ Upgrade to the latest Atomic tree if one is available
 **deploy**
 Download and deploy a specific Atomic tree
 
+**unlock**
+Remove the read-only bind mount on `usr`
+and replace it with a writable overlay filesystem.  This
+default invocation of "unlock" is intended for
+development/testing purposes.  All changes in the overlay
+are lost on reboot (or upgrade).  Pass `--hotfix` to create changes
+that persist on reboot (but still not upgrades).
+
```

Comment:
I am not crazy about the wording.  I was just trying to clarify, but not that important.